ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,72 @@
+Issue Title
+===========
+
+Your issue title is above. Please include the path to the
+topic with the issue. For example, if your issue is that
+our [list of samples](https://stage.docs.microsoft.com/en-us/dotnet/articles/samples-and-tutorials/index)
+is missing a sample, you'd put [samples-and-tutorials/index.md]
+in the issue title.
+
+If your issue is that a topic is missing, please preface
+the issue title with [NewTopic]. That helps us find requests
+for new topics.
+
+General
```

Comment:
I'd prefer keeping the title guidance at the very top. That way it's as close to where a reader would type the new title as possible.

I'm working on making this template much shorter (see @Rick-Anderson 's comments below). I think that will address this comment as well.